### DISCUSSION TITLE
PriceAPI: not working on version 2.2.19

### DISCUSSION THREAD

**Discussion by RPaulsson** [UtilitechAS/amsreader-firmware#582] [Q&A]
![Screenshot 2023-05-30 18 09 32](https://github.com/UtilitechAS/amsreader-firmware/assets/33392437/08f1ddc4-03d0-46b3-842a-91e409201ebf)

How do i get the price to work again ??

**Comment by ArnieO** [comment]
A bug on our server was fixed last evening. If still not OK, reboot the device to force reload of prices.
